<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>120</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Load Image</string>
    </property>
    <widget class="QPushButton" name="loadFolderButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load folder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadImageLButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image_L</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadImageRButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image_R</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>161</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>1. Calibration</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>120</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>1.3 Find extrinsic</string>
     </property>
     <widget class="QPushButton" name="findExtrinsicButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>1.3 Find extrinsic</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="extrinsicSpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>15</number>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="findCornersButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>1.1 Find corners</string>
     </property>
    </widget>
    <widget class="QPushButton" name="findIntrinsicButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>1.2 Find intrinsic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="findDistortionButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>1.4 Find distortion</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showResultButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>1.5 Show result</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>161</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>2. Augmented Reality</string>
    </property>
    <widget class="QLineEdit" name="arTextBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="showWordsOnBoardButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>2.1 show words on board</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showWordsVerticalButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>2.2 show words vertical</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>3. Stereo disparity map</string>
    </property>
    <widget class="QPushButton" name="stereoDisparityMapButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>3.1 stereo disparity map</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>4. SIFT</string>
    </property>
    <widget class="QPushButton" name="loadSiftImage1Button">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadSiftImage2Button">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="keypointsButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>4.1 Keypoints</string>
     </property>
    </widget>
    <widget class="QPushButton" name="matchedKeypointsButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>4.2 Matched Keypoints</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
